src/hotspot/share/classfile/systemDictionaryShared.cpp

```diff
@@ -34,6 +34,7 @@
 #include "cds/dynamicArchive.hpp"
```

Comment:
Pre-existing: I think the include of `cds/archiveHeapLoader.hpp` at line #27 is unnecessary.

Reply:
Removed.